src/starcheck.pl

```diff
@@ -691,11 +696,16 @@ sub json_obsids{
     if (not defined $char_file){
         $out .= "Error.  Characteristics file not found. \n";
     }
+    # dynamic aimpoint files are required after 21-Aug-2016
+    my $AIMPOINT_REQUIRED_AFTER = '2016:234:00:00:00.000';
+    if ((not defined $aimpoint_file) and ($bs[0]->{time} > date2time($AIMPOINT_REQUIRED_AFTER))){
+        $out .= "Error.  dynamic aimpoint file not found. \n";
+    }
```

Comment:
Whoops, I just broke this if-else.  The aimpoint file check should be an independent check before the if-else on the characteristics file.